src/components/admin-header/Header.js

```diff
@@ -7,16 +7,23 @@ import * as BsIcons from "react-icons/bs";
 import * as IoIcons from "react-icons/io";
 import "./Header.css";
 import { useNavigate } from "react-router-dom";
-
-const Header = () => {
+import { getAllNotifications } from "../../module/actions/NotificationAction";
```

Comment:
this should be `import { getAllNotifications } from "../../module/actions/notificationAction";`